user/cinnamon-desktop/template.py

```diff
@@ -0,0 +1,33 @@
+pkgname = "cinnamon-desktop"
+pkgver = "6.2.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = ["-Dalsa=true", "-Dpnp_ids=/usr/share/hwdata/pnp.ids"]
+hostmakedepends = ["gettext", "gobject-introspection", "meson", "pkgconf"]
+makedepends = [
+    "alsa-lib-devel",
+    "elogind-devel",
+    "gdk-pixbuf-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "libpulse-devel",
+    "libx11-devel",
+    "libxext-devel",
+    "libxkbfile-devel",
+    "libxrandr-devel",
+    "xkeyboard-config",
+]
+depends = ["chimera-artwork", "hwdata-pnp"]
+pkgdesc = "Cinnamon desktop library and common settings schemas"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "GPL-2.0-or-later AND LGPL-2.0-or-later"
+url = "https://projects.linuxmint.com/cinnamon"
+source = f"https://github.com/linuxmint/cinnamon-desktop/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "95bb6fc16597601b1febde4bd1317ba9c3ab662119b6b8e602b49e832d3f3eb7"
+# No tests
+options = ["!check"]
```

Comment:
when there's no tests but the default check also passes (like meson just says 'no tests' and continues), you don't have to set this, and just doing nothing is ok